src/shared/bus-util.h

```diff
@@ -49,11 +49,17 @@ struct bus_properties_map {
         size_t offset;
 };
 
+enum {
+        BUS_MAP_COPY_STRING     = 1U << 0, /* If set, strings are copied. Thus, userdata needs to be freed. */
```

Comment:
"COPY" is ambiguous. Why not call this BUS_MAP_STRDUP?

"stored as bool/int" is also not as obvious as it could be. Maybe it's better to be explicit and say "written to a bool/int pointer"?